.storybook/main.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path')
+const webpack = require('webpack')
+module.exports = {
+  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
+  webpackFinal: (config) => {
+    // Necessary to "mock" next/image in Storybook land
+    config.plugins.push(
+      new webpack.DefinePlugin({
```

Comment:
😱

Is this brittle? I panicked a bit when I first saw it but it looks contained specifically to storyboard.

Reply:
Yeah, this is specific to the `Storybook` environment 👍 

`next/image` won't work in the `Storybook` environment. 

`Storybook` use highlights some places where we are coupling the data requests to the UI. For example, `search/stack`. That component will not work as is in `Storybook` as it relies on functions like `loadLesson`, etc. which are reliant on `next`.

But yeah, anything inside the `.storybook` directory won't have an effect on things outside of it 👍